docs/reference/errors-and-warnings/NU1302.md

```diff
@@ -0,0 +1,21 @@
+---
+title: NuGet Error NU1302
+description: NU2302 Error code
+author: Nigusu-Allehu
+ms.date: 07/25/2024
+ms.topic: reference
+f1_keywords: 
+  - "NU1302"
+---
+
+# NuGet Warning NU1302
+
+> You are running the 'restore' operation with an 'HTTP' source: myHttpSource. NuGet requires HTTPS sources. To use an HTTP source, you must explicitly set 'allowInsecureConnections' to true in your NuGet.Config file. Please refer to https://aka.ms/nuget-https-everywhere for more information.
+
+### Issue
+
+`myHttpSource` is an insecure HTTP source. We recommend using HTTPS sources instead.
+
+### Solution
+
+This can be fixed either by removing the HTTP source or disabling HTTP Errors for the specific source by using `allowInsecureConnections` option in your [NuGet config file](../../reference/nuget-config-file.md).

```

Comment:
Should we encourage customers to migrate their sources to HTTPS endpoints?

Reply:
We have that in the error message. I didn't want to repeat it in the solution.